forte/processors/base/batch_processor.py

```diff
@@ -252,6 +257,121 @@ def define_batcher() -> ProcessingBatcher:
         return FixedSizeDataPackBatcher()
 
 
+class Predictor(BaseBatchProcessor):
+    r"""This class is used to perform prediction on features that
+    extracted from the datapack and add the prediction back to the
+    datapack.
+    """
+    @staticmethod
+    def _define_context() -> Type[Annotation]:
```

Comment:
small suggestion, let's place these unused method to the end of the class for the moment?